darwin/future/core/client.py

```diff
@@ -172,7 +172,7 @@ def _generic_call(self, method: Callable, endpoint: str, payload: Optional[dict]
         endpoint = self._sanitize_endpoint(endpoint)
         url = self.config.api_endpoint + endpoint
         if payload is not None:
-            response = method(url, payload)
+            response = method(url, json=payload)
```

Comment:
data didn't work here for some reason

Reply:
I think it only works if the data type isn't "app/json" or on certain methods.